docs/user/00-20-configure-serverless.md

```diff
@@ -0,0 +1,74 @@
+# Overview
+
+Serverless module has its own operator (`serverless-operator`). It watches `Serverless` Custom Resource and re-configure (reconcile) serverless workloads.
+
+Serverless CR becomes the API to configure serverless module. Use it to do the following:
+ - enable/disable internal docker registry
+ - configure external docker registry 
+ - override endpoint for traces collected by serverless functions
+ - override endpoint for eventing
+
+Default configuration of serverless module looks as follows:
+
+```yaml
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: Serverless
+  name: serverless-sample
+spec:
+  dockerRegistry:
+    enableInternal: true
+```
+
+## Configure Docker Registry
+
+By default, Serverless uses PersistentVolume (PV) as the internal registry to store Docker images for Functions. The default storage size of a single volume is 20 GB. This internal registry is suitable for local development.
+
+If you use Serverless for production purposes, it is recommended that you use an external registry, such as Docker Hub, Google Container Registry (GCR), or Azure Container Registry (ACR).
+
+If you want serverless to use external docker registry, create a secret in `kyma-system` Namespace. Such a Secret must contain required data (**username**, **password**, **serverAddress**, and **registryAddress**):
+
+```bash
```

Comment:
Please add three spaces for the command (line 31)

Reply:
done